File: app/src/main/java/com/example/todo/FragmentMe.java
package com.example.todo;

import android.content.Intent;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.example.todo.databinding.FragmentMeBinding;

public class FragmentMe extends Fragment {
    private FragmentMeBinding binding;

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        binding = FragmentMeBinding.inflate(inflater, container, false);
        View viewMe = binding.getRoot();
        binding.fragmentMeAllItemButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(container.getContext(), ActivityAllItem.class);
                startActivity(intent);
            }
        });
        binding.fragmentMeBackgroundSettingButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });
        return viewMe;
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
        binding = null;
    }
}

File: app/src/main/java/com/example/todo/home_fragments/FragmentMe.kt
package com.example.todo.home_fragments

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.example.todo.ActivityAllItem
import com.example.todo.R
import com.example.todo.databinding.FragmentMeBinding
import java.util.concurrent.CountDownLatch

class FragmentMe : FragmentBase() {
    override val title = R.string.main_toolbar_me

    private var _binding: FragmentMeBinding? = null
    private val binding get() = _binding!!

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        _binding = FragmentMeBinding.inflate(inflater, container, false)
        val view: View = binding.root
        binding.fragmentMeAllItemButton.setOnClickListener {
            startActivity(
                Intent(
                    context,
                    ActivityAllItem::class.java
                )
            )
        }
        binding.fragmentMeBackgroundSettingButton.setOnClickListener { }
        return view
    }

    override fun initFragmentData(context: Context, countDownLatch: CountDownLatch) {
        TODO("Not yet implemented")
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }
}